06-building-a-program-start-to-finish/writing-tests/output/test/spec/inverted-index-spec.js

```diff
@@ -0,0 +1,84 @@
+const InvertedIndex = require('../../src/inverted-index.js');
+const samples = require('../samples');
+const FileAPI = require('file-api');
```

Comment:
A base filename should exactly match the name of its default export. [See link](https://github.com/airbnb/javascript#naming--filename-matches-export)